### DISCUSSION TITLE
Update the documented Laravel - PHP Community SDK

### DISCUSSION THREAD

**Discussion by dniccum** [Unleash/unleash#5164] [General]
Hi, I am in the process of getting Unleash stood up on a Laravel application that we have been development on for over 5 years. After reading your documentation, I attempted to utilize the Laravel SDK that was listed on the [official documentation site](https://docs.getunleash.io/reference/sdks#community-sdks). After 30-45 minutes of trial and error, and ultimately never getting it work, I would recommend that this package be removed from the listing and replaced with [this package by JWebb](https://github.com/j-webb/laravel-unleash).

## Reasons

- The documentation for the JWebb composer package is **far better** and is easier to consume than the [laravel-unleash](https://github.com/laravel-unleash/laravel-unleash) package (which is listed on your site).
- The `laravel-unleash` package's error handling is extremely confusing and does not report errors in a clean and digestable manner. The `j-webb` package is superior in this regard.
- I would also argue that the `j-webb` package's syntax with regards to methods and variables is also cleaner.

**Comment by dgorton** [comment]
Thank you for the feedback and sorry for our slow reply! We'll give this a look as well but I wanted to get back to you as soon as I realized that we had missed this discussion. More soon!
